code/game/gamemodes/cult/talisman.dm

```diff
@@ -278,11 +250,104 @@ Rite of Disorientation
+	if(iscultist(user) && iscultist(target))
+		user.drop_item()
+		invoke(target)
+		qdel(src)
+		return
+	..()
+
+//Talisman of Horrors: Breaks the mind of the victim with nightmarish hallucinations
+/obj/item/weapon/paper/talisman/horror
+	cultist_name = "Talisman of Horrors"
+	cultist_desc = "A talisman that will break the mind of the victim with nightmarish hallucinations."
+	color = "#ffb366" // light orange
+	invocation = "Lo'Nab Na'Dm!"
+
+/obj/item/weapon/paper/talisman/horror/attack(mob/living/target, mob/living/user)
+	if(iscultist(user))
+		user.visible_message("<span class='cultitalic'>You disturb [target] with visons of the end!</span>")
+		if(iscarbon(target))
+			var/mob/living/carbon/H = target
+			H.reagents.add_reagent("mindbreaker", 30)
+		qdel(src)
+
+//Talisman of Fabrication: Creates a construct shell out of 25 metal sheets.
+/obj/item/weapon/paper/talisman/construction
+	cultist_name = "Talisman of Construction"
+	cultist_desc = "Use this talisman on at least twenty-five metal sheets to create an empty construct shell"
+	invocation = "Ethra p'ni dedol!"
+	color = "#000000" // black
+
+/obj/item/weapon/paper/talisman/construction/attack_self(mob/living/user)
+	if(iscultist(user))
+		user << "<span class='warning'>To use this talisman, attack the target directly.</span>"
+	else
+		user << "<span class='danger'>There are indecipherable images scrawled on the paper in what looks to be... <i>blood?</i></span>"
+	
+	
+/obj/item/weapon/paper/talisman/construction/attack(obj/M,mob/living/user)
+	if(iscultist(user))
+		user << "<span class='cultitalic'>This talisman will only work on a stack of metal sheets!</span>"
+		log_game("Construct talisman failed - not a valid target")
+
+//Talisman of Shackling: Applies special cuffs directly from the talisman
+/obj/item/weapon/paper/talisman/shackle
+	cultist_name = "Talisman of Shackling"
+	cultist_desc = "Use this talisman on a victim to handcuff them with dark bindings."
+	invocation = "In'totum Lig'abis!"
+	color = "#B27300" // burnt-orange
+	uses = 4
+
+/obj/item/weapon/paper/talisman/shackle/attack(mob/living/target, mob/living/user)
+	if(iscultist(user))
+		if(isrobot(target))
+			..()
+			return
+		if(!isliving(target))
+			user.visible_message("<span class='cultitalic'>This talisman's magic does not affect the dead!</span>")
+			return
+		var/mob/living/L = target
+		if(ishuman(L))
+			CuffAttack(L,user)
+			
+/obj/item/weapon/paper/talisman/shackle/proc/CuffAttack(mob/living/L,mob/living/user)
+	if(!iscarbon(L))
+		return
+	var/mob/living/carbon/C = L
+	if(!C.handcuffed)
+		playsound(loc, 'sound/weapons/cablecuff.ogg', 30, 1, -2)
+		C.visible_message("<span class='danger'>[user] begins restraining [C] with [src]!</span>", \
+								"<span class='userdanger'>[user] begins shaping an dark magic around your wrists!</span>")
+		if(do_mob(user, C, 30))
+			if(!C.handcuffed)
+				C.handcuffed = new /obj/item/weapon/restraints/handcuffs/energy/used(C)
```

Comment:
Instead of this, do;
````
var/obj/item/weapon/restraints/handcuffs/energy/used/Z = new /obj/item/weapon/restraints/handcuffs/energy/used(C)
Z.apply_cuffs(C, user)
````
I'm not sure what, if anything, bad would happen if you apply the cuffs directly, but it's better to use the proper proc for it.